demo/schema.py

```diff
@@ -319,6 +327,23 @@ def create_project(
             ),
         )
 
+    @gql.django.input_mutation
+    def create_quiz(
+        self,
+        info: Info,
+        title: str,
+        full_clean_options: bool = False
+    ) -> QuizType:
+        return cast(
+            QuizType,
+            resolvers.create(
+                info,
+                Quiz,
+                {"title": title},
+                full_clean=FullCleanOptions(exclude=["sequence"]) if full_clean_options else True
```

Comment:
The idea of the typeddict is to pass a dict directly, and the type checker will check that it conforms to the definition. DOn't need to import it and use it here.

I would not reject the PR for this but I know that there's a pep8 issue that you will have to fix in another file as well

```suggestion
                full_clean={"exclude": ["sequence"]} if full_clean_options else True
```

Reply:
fixed all ci issues by pyright, code lint issues.
sorry for bothering you!